### DISCUSSION TITLE
How to input {} and []?

### DISCUSSION THREAD

**Discussion by marc2o** [lapce/lapce#571] [Q&A]
I’m on an Intel MacBook Air (13", Early 2015) running macOS Monterey 12.4 (21F79). When using Lapce it is not possible to input curly braces `{ }`, square brackets `[ ]` or really any character that requires the Alt key. How is it done? Since I can’t find this as a bug in the issues section, I surmise that I am »holding it wrong«. :c)

Edit: I’m on a German system, if that helps. The problem persists in the new version 0.1.2.

**Comment by muraca** [comment]
As of [commit fe4e684](https://github.com/lapce/lapce/commit/fe4e684821ea4a2bd206464d519c4ecc22b0c1a3), I also have this problem, on an Italian system.
